src/features/pages/deriv-ez/deprecated-other-platforms/index.tsx

```diff
@@ -41,10 +48,14 @@ const OurPlatforms = () => {
     ]
     return (
         <>
-            <GenericCarousel
-                renderableData={carouselData}
-                mainHeading="_t_Check out our other platforms_t_"
-            />
+            {is_loaded ? (
+                <GenericCarousel
+                    renderableData={carouselData}
+                    mainHeading="_t_Check out our other platforms_t_"
+                />
+            ) : (
+                <></>
```

Comment:
```suggestion
    null
```

Reply:
not sure shall we change to null instead of empty fragments?

Reply:
Fixed